aodntools/timeseries_products/hourly_timeseries.py

```diff
@@ -337,7 +337,7 @@ def generate_netcdf_output_filename(nc, facility_code, data_code, site_code, pro
 
 
 
-def write_netCDF_aggfile(nc_aggregated, ncout_filename, encoding, file_path):
+def write_netCDF_aggfile(nc_aggregated, ncout_filename, encoding):
     """
     write netcdf file
     :param file_path: path where to write the file
```

Comment:
Might as well update the doc string to be consistent with the function arguments...

Reply:
done